### DISCUSSION TITLE
Init from starter returns - Cannot connect to [statamic.com] to validate license!

### DISCUSSION THREAD

**Discussion by Galanthus** [statamic/cms#9812] [Troubleshooting] [answered]
Trying to run a project from scratch on a windows 11 machine and I am getting the following error on init:
**Cannot connect to [statamic.com] to validate license!**

Any idea?

**Comment by duncanmcclean** [comment, marked as answer]
I assume you've already checked you can view Statamic.com in your browser?

Sometimes with Windows, SSL certificates can cause issues with the Statamic CLI. It might be worth following [this guide](https://web.archive.org/web/20220126073217/https://websolutionstuff.medium.com/how-to-fix-curl-error-60-ssl-certificate-problem-246064e107d3) to see if it helps. _(I've linked to an archived version of the article since it's been taken down)_

If that doesn't fix the issue, are there any other errors showing or is that the only output of the `statamic new ...` command?

**Reply by Galanthus** [reply]
When I `statamic new project` it works but running from a free starter like the Peak I get this error.

**Reply by Galanthus** [reply]
That CURL issue fixed id :D thanks!

**Reply by duncanmcclean** [reply]
Awesome, glad to hear it! 🎉
